src/containers/search/SearchBar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { SearchButton, SearchInput } from '../../components';
+
+const SearchBarStyled = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin-top: 15px;
+`;
+
+const SearchBar = () => (
+  <SearchBarStyled>
+    <SearchInput></SearchInput>
```

Comment:
If you don't pass any child components inside parent. It could be written as `<SearchInput />`
The same for next line